### PR TITLE
Me maine gov normalize  add area codes

### LINKED ISSUE #236: Put phone numbers w/o area codes into the notes section in me/maine_gov normalizer
Some phone numbers from the `me/maine_gov` don't have area codes. Right now, if this happens, the phone number just isn't included. 

It'd be great if, instead of being removed, phone numbers without area codes were included in the notes.

### PR DESCRIPTION
**Pull request by rossry**
# normalize maine_gov for me (improve phone numbers)

| Key Details |
|-|
| Resolves #236 |
State: me |
Site: maine_gov |

## Notes
* Area code 207 is the only area code in Maine, so let's assume that when given a 7-digit phone number

## Data sample
```json
{"id": "me_maine_gov:bridgton_hospital_bridgton", "name": "Bridgton Hospital", "address": null, "location": null, "contact": [{"contact_type": "general", "phone": "(207) 520-2917", "website": null, "email": null, "other": null}, {"contact_type": "general", "phone": null, "website": "https://www.cmhc.org/about-us/cmh-alert/vaccine-covid-19/", "email": null, "other": null}, {"contact_type": "booking", "phone": "(207) 520-2917", "website": null, "email": null, "other": null}, {"contact_type": "booking", "phone": null, "website": "https://www.cmhc.org/about-us/cmh-alert/vaccine-covid-19/", "email": null, "other": null}, {"contact_type": "booking", "phone": null, "website": null, "email": null, "other": "Call <a href=\"tel:207-520-2917\">207-520-2917</a> to make an appointment. <a href=\"https://www.cmhc.org/about-us/cmh-alert/vaccine-covid-19/\">More information here.</a>"}], "languages": null, "opening_dates": null, "opening_hours": null, "availability": null, "inventory": null, "access": null, "parent_organization": null, "links": null, "notes": ["city:Bridgton", "county:Cumberland", "audience:Public"], "active": null, "source": {"source": "me_maine_gov", "id": "bridgton_hospital_bridgton", "fetched_from_uri": "https://www.maine.gov/covid19/vaccines/vaccination-sites", "fetched_at": "2021-05-01T10:43:04.956954", "published_at": null, "data": {"providerName": "Bridgton Hospital", "city": "Bridgton", "county": "Cumberland", "schedulingInfo": "Call <a href=\"tel:207-520-2917\">207-520-2917</a> to make an appointment. <a href=\"https://www.cmhc.org/about-us/cmh-alert/vaccine-covid-19/\">More information here.</a>", "minimumAge": "", "audience": "Public", "phoneNumber": ["207-520-2917"], "website": ["https://www.cmhc.org/about-us/cmh-alert/vaccine-covid-19/"]}}}
{"id": "me_maine_gov:bucksport_regional_health_center_bucksport", "name": "Bucksport Regional Health Center", "address": null, "location": null, "contact": [{"contact_type": "booking", "phone": null, "website": null, "email": null, "other": "By appointment only. Patients will be notified of eligibility to receive the vaccine."}], "languages": null, "opening_dates": null, "opening_hours": null, "availability": null, "inventory": null, "access": null, "parent_organization": null, "links": null, "notes": ["city:Bucksport", "county:Hancock", "audience:Patient Only"], "active": null, "source": {"source": "me_maine_gov", "id": "bucksport_regional_health_center_bucksport", "fetched_from_uri": "https://www.maine.gov/covid19/vaccines/vaccination-sites", "fetched_at": "2021-05-01T10:43:04.956954", "published_at": null, "data": {"providerName": "Bucksport Regional Health Center", "city": "Bucksport", "county": "Hancock", "schedulingInfo": "By appointment only. Patients will be notified of eligibility to receive the vaccine.", "minimumAge": "", "audience": "Patient Only", "phoneNumber": [], "website": []}}}
{"id": "me_maine_gov:b_street_clinic_lewiston", "name": "B Street Clinic", "address": null, "location": null, "contact": [{"contact_type": "general", "phone": null, "website": "http://www.communityclinicalservices.com/covid-19-vaccines/", "email": null, "other": null}, {"contact_type": "booking", "phone": null, "website": "http://www.communityclinicalservices.com/covid-19-vaccines/", "email": null, "other": null}, {"contact_type": "booking", "phone": null, "website": null, "email": null, "other": "To make an appointment visit <a href=\"http://www.communityclinicalservices.com/covid-19-vaccines/\">http://www.communityclinicalservices.com/covid-19-vaccines/</a>."}], "languages": null, "opening_dates": null, "opening_hours": null, "availability": null, "inventory": null, "access": null, "parent_organization": null, "links": null, "notes": ["city:Lewiston", "county:Androscoggin", "audience:Public"], "active": null, "source": {"source": "me_maine_gov", "id": "b_street_clinic_lewiston", "fetched_from_uri": "https://www.maine.gov/covid19/vaccines/vaccination-sites", "fetched_at": "2021-05-01T10:43:04.956954", "published_at": null, "data": {"providerName": "B Street Clinic", "city": "Lewiston", "county": "Androscoggin", "schedulingInfo": "To make an appointment visit <a href=\"http://www.communityclinicalservices.com/covid-19-vaccines/\">http://www.communityclinicalservices.com/covid-19-vaccines/</a>.", "minimumAge": "", "audience": "Public", "phoneNumber": [], "website": ["http://www.communityclinicalservices.com/covid-19-vaccines/"]}}}
{"id": "me_maine_gov:calais_regional_hospital_calais", "name": "Calais Regional Hospital", "address": null, "location": null, "contact": [{"contact_type": "general", "phone": null, "website": "https://www.calaishospital.org/covid-19/covid-19-vaccination/covid-19-vaccine-appointment-registration-page/", "email": null, "other": null}, {"contact_type": "booking", "phone": null, "website": "https://www.calaishospital.org/covid-19/covid-19-vaccination/covid-19-vaccine-appointment-registration-page/", "email": null, "other": null}, {"contact_type": "booking", "phone": null, "website": null, "email": null, "other": "<p>Visit <a href=\"https://www.calaishospital.org/covid-19/covid-19-vaccination/covid-19-vaccine-appointment-registration-page/\">www.calaishospital.org/covid-19/</a> to register for an appointment.</p>"}], "languages": null, "opening_dates": null, "opening_hours": null, "availability": null, "inventory": null, "access": null, "parent_organization": null, "links": null, "notes": ["city:Calais", "county:Washington County", "audience:Public"], "active": null, "source": {"source": "me_maine_gov", "id": "calais_regional_hospital_calais", "fetched_from_uri": "https://www.maine.gov/covid19/vaccines/vaccination-sites", "fetched_at": "2021-05-01T10:43:04.956954", "published_at": null, "data": {"providerName": "Calais Regional Hospital", "city": "Calais", "county": "Washington County", "schedulingInfo": "\n<p>Visit <a href=\"https://www.calaishospital.org/covid-19/covid-19-vaccination/covid-19-vaccine-appointment-registration-page/\">www.calaishospital.org/covid-19/</a> to register for an appointment.</p>\n", "minimumAge": "", "audience": "Public", "phoneNumber": [], "website": ["https://www.calaishospital.org/covid-19/covid-19-vaccination/covid-19-vaccine-appointment-registration-page/"]}}}
{"id": "me_maine_gov:caribou_health_center_caribou", "name": "Caribou Health Center", "address": null, "location": null, "contact": [{"contact_type": "general", "phone": "(207) 498-1638", "website": null, "email": null, "other": null}, {"contact_type": "general", "phone": null, "website": "http://www.pineshealth.org/", "email": null, "other": null}, {"contact_type": "booking", "phone": "(207) 498-1638", "website": null, "email": null, "other": null}, {"contact_type": "booking", "phone": null, "website": "http://www.pineshealth.org/", "email": null, "other": null}, {"contact_type": "booking", "phone": null, "website": null, "email": null, "other": "Information on vaccination clinics provided by Pines Health Services is available by calling <a href=\"tel:207-498-1638\">207-498-1638</a> or visiting <a href=\"http://www.pineshealth.org/\">www.pineshealth.org/</a>."}], "languages": null, "opening_dates": null, "opening_hours": null, "availability": null, "inventory": null, "access": null, "parent_organization": null, "links": null, "notes": ["city:Caribou", "county:Aroostook", "audience:Patient Only"], "active": null, "source": {"source": "me_maine_gov", "id": "caribou_health_center_caribou", "fetched_from_uri": "https://www.maine.gov/covid19/vaccines/vaccination-sites", "fetched_at": "2021-05-01T10:43:04.956954", "published_at": null, "data": {"providerName": "Caribou Health Center", "city": "Caribou", "county": "Aroostook", "schedulingInfo": "Information on vaccination clinics provided by Pines Health Services is available by calling <a href=\"tel:207-498-1638\">207-498-1638</a> or visiting <a href=\"http://www.pineshealth.org/\">www.pineshealth.org/</a>.", "minimumAge": "", "audience": "Patient Only", "phoneNumber": ["207-498-1638"], "website": ["http://www.pineshealth.org/"]}}}
{"id": "me_maine_gov:cary_medical_center_caribou", "name": "Cary Medical Center", "address": null, "location": null, "contact": [{"contact_type": "general", "phone": "(207) 498-1638", "website": null, "email": null, "other": null}, {"contact_type": "general", "phone": null, "website": "http://www.carymedicalcenter.org/covid-19/", "email": null, "other": null}, {"contact_type": "booking", "phone": "(207) 498-1638", "website": null, "email": null, "other": null}, {"contact_type": "booking", "phone": null, "website": "http://www.carymedicalcenter.org/covid-19/", "email": null, "other": null}, {"contact_type": "booking", "phone": null, "website": null, "email": null, "other": "Call <a href=\"tel:207-498-1638\">207-498-1638</a> or visit<a href=\"http://www.carymedicalcenter.org/covid-19/\">www.carymedicalcenter.org/covid-19/</a> to make an appointment."}], "languages": null, "opening_dates": null, "opening_hours": null, "availability": null, "inventory": null, "access": null, "parent_organization": null, "links": null, "notes": ["city:Caribou", "county:Aroostook", "audience:Public"], "active": null, "source": {"source": "me_maine_gov", "id": "cary_medical_center_caribou", "fetched_from_uri": "https://www.maine.gov/covid19/vaccines/vaccination-sites", "fetched_at": "2021-05-01T10:43:04.956954", "published_at": null, "data": {"providerName": "Cary Medical Center", "city": "Caribou", "county": "Aroostook", "schedulingInfo": "Call <a href=\"tel:207-498-1638\">207-498-1638</a> or visit<a href=\"http://www.carymedicalcenter.org/covid-19/\">www.carymedicalcenter.org/covid-19/</a> to make an appointment.", "minimumAge": "", "audience": "Public", "phoneNumber": ["207-498-1638"], "website": ["http://www.carymedicalcenter.org/covid-19/"]}}}
```

## Before Opening a PR
- [x] I tested this using the CLI (e.g., `poetry run vaccine-feed-ingest <state>/<site>`)
- [x] I ran auto-formatting: `poetry run tox -e lint-fix`

### CODE REVIEW

**Comment by eliblock** [comment]
Hi! We've added significant validation to the schema, and want to make sure this normalizer takes advantage of it.

Could you pull in the latest `main` and run your code on top of it? Full details on what changed and why are in #278

**Comment by rossry** [comment]
@eliblock -- now passing against new validation.

File: vaccine_feed_ingest/runners/me/maine_gov/normalize.py [hunk 1]
```diff
@@ -73,7 +80,7 @@ def _get_contacts(site: dict) -> List[schema.Contact]:
 
     website_matches = re.search('href="(http.*)"', scheduling_info_raw)
     if website_matches:
-        website = website_matches.group(1)
+        website = website_matches.group(1).split(" ")[0]
```
Comment (eliblock) [inline vaccine_feed_ingest/runners/me/maine_gov/normalize.py:83]: Are there sometimes multiple websites split by spaces? If so - can we add `Contact` entries for each of them?

**Review by eliblock — APPROVED** [review]
Thank you for updating! I'm going to merge this, but left one question for a potential followup.

**Pull request merged**